pom.xml

```diff
@@ -335,6 +335,7 @@
     </profiles>
     <properties>
         <vertx.version>4.2.1</vertx.version>
+        <vertx-redis-client.version>4.3.3</vertx-redis-client.version>
```

Comment:
```xml
    <vertx.version>4.2.1</vertx.version>
    <vertx-redis-client.version>4.3.3</vertx-redis-client.version>
```
I assume it is correct to use different versions for those two?

Reply:
we intend to update the redis client only, should be legitimate I think, it does compile and come up on my local system